src/engraving/tests/parts_data/part-54346-parts.mscx

```diff
@@ -337,21 +335,6 @@
           </Instrument>
         </Part>
       <Staff id="1">
-        <VBox>
```

Comment:
Looks like here and in other `parts` tests an entire VBox disappeared. Is that expected?

Reply:
Yep, that's by design: by default now boxes are _not_ included in parts